src/redux/reducer.js

```diff
@@ -0,0 +1,23 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+export const getGreetings = createAsyncThunk('greeting/getGreetings', async () => {
+  const response = await fetch('http://localhost:3000/api/greetings/random');
```

Comment:
- [ ] I am not able to fetch greetings on the frontend that is probably because you did not add `cors` to the backend please fix that
<img width="697" alt="Screenshot 2023-07-14 at 16 08 41" src="https://github.com/Balikuddembe/hello-react-front-end/assets/44745855/202324cd-6075-4b10-87e3-554563e7c59e">